src/styles/components/_hero_section.scss

```diff
@@ -0,0 +1,50 @@
+//
+// Variables
+//
+
+@import "base/variables";
```

Comment:
do we normally import them here? I think having it in the root styles.scss should be enough before all other imports.